### ISSUE TITLE
Simplify the `__init__` of the `MyScaler` class

### ISSUE THREAD

**Issue by pierrePalud** [pierrePalud/beetroots#3]
Currently, the `__init__` method of the `MyScaler` class is written to either 
- take a grid in input and use it to compute the `mean_` and `std_` vectors (used in the normalization) -- this approach currently raises an `NotImplementedError()`.
- take directly the `mean_` and `std_` vectors in input.

In the rest of the code, the `mean_` and `std_` are in practice loaded from a `sklearn.preprocessing.StandardScaler()` object and then given to initialize the `MyScaler` instance, which seems needlessly complicated.

To clarify the code, I suggest:
- [ ] to remove the option of providing a grid to the `__init__` (which in any case is currently not supported)
- [ ] to indicate the `mean_` and `std_` vectors in the `input_params.yaml` file, just like the `list_is_log` input.

The second point can help to remove the distinction between the scaling factor `kappa` and the other physical parameters -- see #1. When `kappa` is considered, the `input_params.yaml` file could contain, eg, 

```yaml
forward_model:
  ...
  fixed_params: # must contain all the params in list_names of the Simulation object. Values are in linear scale.
    kappa: null
    another_param_in_linscale: null
    another_param_in_logscale: null
    another_param_fixed_param: 1.0
  is_log_scale_params: # defines the scale to work with for each param (either log or lin)
    kappa: True
    another_param_in_linscale: False
    another_param_in_logscale: True
    another_param_fixed_param: True
  mean_scale_params: # defines the mean for the normalization
    kappa: 0.0
    another_param_in_linscale: (mean of a dataset in linscale, or set manually to eg the mean of the validity interval)
    another_param_in_logscale: (mean of a dataset in logscale, or set manually to eg the mean of the validity interval (in log))
    another_param_fixed_param: (mean of a dataset in logscale, or set manually to eg the mean of the validity interval (in log))
  std_scale_params: # defines the standard deviation (std) for the normalization
    kappa: 1.0
    another_param_in_linscale: (std of a dataset in linscale, or set manually to eg the validity interval size)
    another_param_in_logscale: (std of a dataset in logscale, or set manually to eg the validity interval size (in log))
    another_param_fixed_param:  (std of a dataset in logscale, or set manually to eg the validity interval size (in log))
```

Notes:
- on `std_scale_params` value for the scaling factor: one could choose 1.0 by default, or use another value so that the scaling factor has a similar validity interval size as the other physical parameters in the sampled space.
- on the mean and std if set manually (and if the forward model is an instance of the `NeuralNetworkApprox` class): they must match the normalization applied to data used for the training.

**Comment by pierrePalud** [comment]
I removed the option, as described above.

In addition, to avoid handling the scaling factor `kappa` as a special case in `_from_scaled_to_lin` and `_from_lin_to_scaled`, I changed the `self.mean_` and `self.std_` numpy arrays, such that if there is a `kappa` in the physical parameters, it should be sampled in log scale. Thus I set 
- its mean to 0 
- its std to `1 / np.log(10)`.

This value of the std was chosen for the common validity interval on kappa, [0.1, 10], such that in scaled space it becomes [-e, e], that is, approximately [-2.7, 2.7]. This scaled interval has a similar size as the scaled validity intervals for the other physical parameters (that are all normalized).
